converter/internal/promtailconvert/testdata/unsupported.diags

```diff
@@ -14,5 +14,4 @@
 (Error) unsupported service discovery marathon was provided
 (Error) unsupported service discovery nerve was provided
 (Error) unsupported service discovery openstack was provided
-(Error) unsupported service discovery serverset was provided
-(Error) unsupported service discovery triton was provided
\ No newline at end of file
+(Error) unsupported service discovery serverset was provided
```

Comment:
Off-topic question. One of the discovery mechanisms I'm doing (`discovery.kuma`) does not have an entry here. Is it just an omission?